lms/static/sass/discussion/views/_home.scss

```diff
@@ -0,0 +1,166 @@
+// discussion - home layout
+// ====================
+
+// home - layout
+.container .discussion-body {
+
+  .blank-slate {
+      padding: 40px 40px 10px;
```

Comment:
Absolutely. I can make all the _home.scss partial changes you've suggested in a separate fixup commit, to help make clear the move changes (current) and the cleanup changes (fixup hotness.)